src/app/containers/header/header.component.scss

```diff
@@ -0,0 +1,10 @@
+header {
+    padding-bottom: 50px;
```

Comment:
Hi @sakanam I've checked your branch out locally, one thing I've noticed is the feedback bar clashes with the cookie banner when that's active due to the absolute positioning used in app-phase-banner rule. You could add `position: relative` rule here to prevent that. Let me know if your want me to send you a screenshot

Reply:
added relative positioning to header, fixing clash with cookies banner